common/static/common/js/karma.common.conf.js

```diff
@@ -0,0 +1,216 @@
+// Common configuration for Karma
+/* jshint node: true */
+'use strict';
```

Comment:
Thanks, I hadn't realized that! I see a number of JSHint errors in that file which I thought were related to this, but it seems not. Can you run the linter against your files and fix any issues that crop up. Thanks.